.github/workflows/velox_backend.yml

```diff
@@ -105,11 +105,15 @@ jobs:
       - uses: actions/upload-artifact@v4
         with:
           name: velox-native-lib-centos-7-${{github.sha}}
-          path: ./cpp/build/releases/
+          path: ｜
+            ./cpp/build/releases/libgluten.*
```

Comment:
This job will fail if the artifact files do not exist. 

https://github.com/apache/incubator-gluten/actions/runs/13897311029/job/38880793037?pr=9028